environment/env_default.go

```diff
@@ -1,19 +0,0 @@
-package environment
-
-// DefaultEnv is supposed to contain all default
-// .env file injected upon build time
-const DefaultEnv string = `
-# Only print out the generated commands instead of running them
-# KOOL_DEBUG=0
-# Prints out all commands and their output
-# KOOL_VERBOSE=0
-
-# Optional, default is folder name
-# KOOL_NAME=custom_name
-
-# Host user for mapping execution within the containers using kool images
-# KOOL_ASUSER=$UID
```

Comment:
@fabriciojs @dbpolito - does the elimination of these defaults require us to change the Docs in any way (as far as how users set up their `.env` file)?

Reply:
@indiegenus it does not! all of those defaults were commented (` # `), they actually had no effect at all.